lib/screenplay/v3_api.ex

```diff
@@ -51,6 +47,23 @@ defmodule Screenplay.V3Api do
     end
   end
 
+  defp do_fetch(url, headers, opts) do
+    retry with: Stream.take(constant_backoff(500), 3) do
+      HTTPoison.get(
+        url,
+        headers,
+        Keyword.merge(@default_opts, opts)
+      )
+    after
+      result -> result
```

Comment:
It seems like this is the default and can be omitted.

https://hexdocs.pm/retry/Retry.html#retry/2
> The `after` and `else` clauses are optional. By default, a successful value is just returned.